app/scrapers/sections_scraper.rb

```diff
@@ -15,6 +15,7 @@
 # announce connection and connect
 puts "Connecting to #{host}:#{port}"
 db = MongoClient.new(host, port, pool_size: 2, pool_timeout: 2).db('umdclass')
+# profs_db = MongoClient.new(host, port, pool_size: 2, pool_timeout: 2).db('umdprof')
```

Comment:
thanks for keeping this option open. I'd say use your judgement and either use a profs db or just keep everything in the `umdclass` db. I think I favor the latter but haven't thought too much about it. no need to commit commented-out code